lib/svg-sprite/shape.js

```diff
@@ -341,9 +341,7 @@ SVGShape.prototype._initSVG = function() {
 
     if (entities) {
         let svg = this.svg.current.substr(svgStart.index);
-        for (entity in entityMap) {
-            svg = svg.replace(`&${entity};`, entityMap[entity]);
-        }
+        svg = Object.entries(entityMap).map((key, value) => svg.replace(`&${key};`, value));
```

Comment:
1. svg now is an array instead of string
2. `reduce` seems to be more obvious

Reply:
You are right of course, didn't check it a lot. I'll skip this for now since there's another place too.